packages/www/src/components/Button.js

```diff
@@ -0,0 +1,28 @@
+/** @jsx jsx */
+import { jsx } from 'theme-ui'
+
+export const buttonStyles = {
+  fontSize: '16px',
```

Comment:
seems like some of these values could come from the theme. `fontSize: 2, borderRadius: 1` kind of stuff. Especially since this is a specific-height button though I'm not too concerned about it. Just a thought.

Reply:
This is a good callout. It's my first time using theme-ui so there might be a few more of these in the future. Will clean this up in a future PR, thanks!